tools/advisor/advisor/rule_parser_example.py

```diff
@@ -0,0 +1,74 @@
+# Copyright (c) 2011-present, Facebook, Inc.  All rights reserved.
+#  This source code is licensed under both the GPLv2 (found in the
+#  COPYING file in the root directory) and Apache 2.0 License
+#  (found in the LICENSE.Apache file in the root directory).
+
+import argparse
+from advisor.db_log_parser import DatabaseLogs
+from advisor.db_options_parser import DatabaseOptions
+from advisor.db_stats_fetcher import OdsStatsFetcher
+import pickle
+from advisor.rule_parser import RulesSpec
+
+
+def main(args):
```

Comment:
Is this a test file? If yes, can we rename it?

Reply:
I used this file to run and manually test the Rules parser when that was the only code. I am not using it at all now, so I am deleting the file.